ttnn/cpp/ttnn/tensor/layout/page_config.cpp

```diff
@@ -64,13 +64,13 @@ bool PageConfig::is_row_major() const {
     return std::holds_alternative<RowMajorPageConfig>(config_);
 }
 
-std::optional<Tile> PageConfig::get_tile() const
+Tile PageConfig::get_tile() const
 {
     if(std::holds_alternative<TilePageConfig>(config_)) {
         return std::get<TilePageConfig>(config_).get_tile();
     }
 
-    return std::nullopt;
+    return Tile();
```

Comment:
would like to discuss why

Reply:
This code changed, added a comment and a deprecation warning related to this

Reply:
also wondering about what it means to return default constructed tile here?

Reply:
This code is no longer here. It was replaced with storing Tile in RowMajor with a comment:
```
 // This is currently needed for compatibility reasons.
 // Each time tile is specified, a warning will be issued. This should be removed soon.
```